### ISSUE TITLE
[GENERIC] Some chart improvements and suggested metrics

### ISSUE THREAD

**Issue by chuxinh** [andrewhong5297/Crypto-Grants-Analysis#3]
## Show grants/funding consistency
Would like a way to show funding consistency across ecosystem, in addition to the high level summary on the top. An example would be the following:
![image](https://github.com/andrewhong5297/Crypto-Grants-Analysis/assets/30974572/e4f75700-960b-495b-95fc-22fde15a0c93)
Query: [link](https://dune.com/queries/3415677/5734903)

## Hard to view progress and effectiveness of grantees
Instead of showing the breakdown, maybe we can have grantee onchain performance on aggregate and their impact on the rest of the ecosystem instead? 

I personally quite like the view we had in [Optimism - Incentive Program Performance Summary]. Although this would require us knowing when each protocol actually started implementing any incentive/rewards. A proxy would be comparing onchain metrics before grant date, and last 30 days to see any incremental improvement overtime 

(https://dune.com/oplabspbc/optimism-incentive-program-usage-summary) 
![image](https://github.com/andrewhong5297/Crypto-Grants-Analysis/assets/30974572/b48a5cb4-5dd3-4611-8dda-f37de28d2eb0)

## Create a view to see project overlaps 
To answer questions such as if projects been getting grants from various different protocols, what are they, how much have the received, and how successful they are

## Create a view to understand the vertical of grants allocation
This can possibly help us understand what's important to a protocol, and what vertical of the ecosystem they're trying to invest in. We can agree on some pre-defined verticals that can be applied across the projects

**Comment by andrewhong5297** [comment]
1. I just added this chart, thanks for building it!
2. Yeah this probably gets divisive. My thought process has more been "how does protocol usage track with grant size", while we take the time to figure out how to identify "when" a protocol started and stopped implementing rewards. There isn't really a standard for this, and sometimes grants go to covering infra costs and not to distributing incentives to users. Open to discussing this further
3. Yup, Rabbithole has good charts here in their [STIP dashboard](https://dune.com/boost_xyz/stip).
![image](https://github.com/andrewhong5297/Crypto-Grants-Analysis/assets/47720952/cfec990e-3ea8-439a-94fe-e54d422c5603). I think for these, we will need to manually label "domains". Also a larger discussion, I think once we get all the basic grants data in with dune namespaces then we can go back through it.
4. I think related to 3, "domains" should help us identify verticals as well.

**Comment by MSilb7** [comment]
> Show grants/funding consistency

Similarly here, I'm thinking of some way to show like "what is the distribution rate 'right now'." It could be like a trailing 6 month average & monthly growth rate?

**Comment by andrewhong5297** [comment]
> > Show grants/funding consistency
> 
> 
> 
> Similarly here, I'm thinking of some way to show like "what is the distribution rate 'right now'." It could be like a trailing 6 month average & monthly growth rate?

Yeah the moving average seems simple enough to add to the chart. Not sure what the growth rate tells us? Would it be growth rate of number of grants or purely usd amounts?
